docs/blog/crucial-tests/index.md

```diff
@@ -394,6 +392,50 @@ beforeAll(() => {
 });
 ```
 
+## 📦 Test the package as a consumer
+
+a lot of times we don't test our packages as how our consumers will use them,
```

Comment:
Not the same format like other bullets, e.g., "What & why:" section

Reply:
thanks, fixed